asllib/Typing.ml

```diff
@@ -102,7 +102,7 @@ let slices_width env =
   let minus = binop MINUS in
   let one = !$1 in
   let slice_width = function
-    | Slice_Single _ -> one
+    | Slice_Arg _ -> one
```

Comment:
How come a `Slice_Arg` has width if it's not a bitvector slice? Unless this is only for ASLv0?

Reply:
This is for compatibility with ASLv0. Since `Slice_Single` is desugared at parsing in ASLv1, there has to be something emitted for v0 parsing.